### ISSUE TITLE
Addressing Global Search Aspect of 508-defect-2 [SCREENREADER]: Search fields should have the landmark role="search"

### ISSUE THREAD

**Issue by denisecoveyduc** [department-of-veterans-affairs/va.gov-team#26506]
##  Description
As a part of the staging review for UI Consistency for Global Search #26360 was created. There are two parts, one for search and discovery and the other for the public websites team. This ticket is to resolve the search and discovery portion for global search pages.

## Background/Context

- GH #26360

## Tasks
- [x] Search fields should have the landmark role="search"

## Acceptance Criteria
- [ ] Review and acknowledge feedback.
- [ ] Fix and/or document decisions made.
- [ ] Notify Accessibility specialist that the issue has been resolved so they can validate the fix.
